CLI pages for review (#465)
newrelic/developer-website: src/markdown-pages/nr1-cli/nr1-config.mdx

---
path: '/explore-docs/nr1-config'
title: 'New Relic One CLI config commands'
template: 'GuideTemplate'
description: 'An overview of the commands you can use to configure your New Relic One CLI preferences.'
tileShorthand:
  title: 'Config CLI commands'
  description: 'Configure your New Relic One CLI preferences.'
redirects:
---

<Intro>

> To configure your New Relic One CLI preferences, use the commands below. You can click any command to see its usage and options, among other details.

Reviewer: ```suggestion
To configure your New Relic One CLI preferences, use the commands below. You can click any command to see it's usage options and additional details about the command.
```

Reviewer 2: s/b "its"
